docs/README.md

```diff
@@ -293,16 +293,26 @@ Optional:
 
 ```sh
 # Install latest Go compiler
-sudo add-apt-repository ppa:longsleep/golang-backports 
+sudo add-apt-repository ppa:longsleep/golang-backports
 sudo apt update
 sudo apt install golang-go
 
 # Install lib dependencies
 sudo apt install libbrotli-dev liblzo2-dev libsodium-dev curl cmake
 
 # Fetch project and build
+# Go 1.15 and below
 go get github.com/wal-g/wal-g
-cd ~/go/src/github.com/wal-g/wal-g
+# Go 1.16+ - just clone repository to $GOPATH
+git clone https://github.com/wal-g/wal-g $(go env GOPATH)/src/github.com/wal-g/wal-g
```

Comment:
Cloning all repo seems reasonable, since we need it for development. However i believe it would be better to mention `--single-branch --depth=1` options. Something like `# if you want to save space add --depth=1 or --single-branch`

Reply:
Updated